### PR TITLE
LanguagePicker: Use English language names for search and tooltips

### PR DESCRIPTION
**Pull request by jsnajdr**
Adds a module with English names for all 137 languages. Until now, `config/_shared.json` knows only the autonym names.

Use it for showing a tooltip over a language in the `LanguagePickerModal` and for search.

Open questions:
- how to approach translating these 137 language names? Just make them of the standard translation file (`widgets.wp.com/languages/calypso/${langSlug}.json`)? Or some independent file?
- consider loading the `language-names.js` module with `asyncRequire`. Or maybe the whole `LanguagePickerModal` could be inside an `AsyncLoad` -- it's quite big already and rarely used.
- improve search by making it ignore diacritics (i.e., find "čeština" when searching for "cestina").

### CODE REVIEW

File: client/components/language-picker/language-names.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+	'uz': translate => translate( 'Uzbek' ),
+	'vec': translate => translate( 'Venetian' ),
+	'vi': translate => translate( 'Vietnamese' ),
+	'wa': translate => translate( 'Walloon' ),
+	'xal': translate => translate( 'Kalmyk' ),
+	'yi': translate => translate( 'Yiddish' ),
+	'yo': translate => translate( 'Yoruba' ),
+	'za': translate => translate( 'Zhuang' ),
+	'zh-cn': translate => translate( 'Simplified Chinese' ),
+	'zh-tw': translate => translate( 'Traditional Chinese' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Traditional', { context: 'Theme Showcase term name'} )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/language-picker-english-lang-names)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+	'tr': translate => translate( 'Turkish' ),
+	'tt': translate => translate( 'Tatar' ),
+	'ty': translate => translate( 'Tahitian' ),
+	'udm': translate => translate( 'Udmurt' ),
+	'ug': translate => translate( 'Uyghur' ),
+	'uk': translate => translate( 'Ukrainian' ),
+	'ur': translate => translate( 'Urdu' ),
+	'uz': translate => translate( 'Uzbek' ),
+	'vec': translate => translate( 'Venetian' ),
+	'vi': translate => translate( 'Vietnamese' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Vietnam' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+	'tir': translate => translate( 'Tigrinya' ),
+	'tl': translate => translate( 'Tagalog' ),
+	'tr': translate => translate( 'Turkish' ),
+	'tt': translate => translate( 'Tatar' ),
+	'ty': translate => translate( 'Tahitian' ),
+	'udm': translate => translate( 'Udmurt' ),
+	'ug': translate => translate( 'Uyghur' ),
+	'uk': translate => translate( 'Ukrainian' ),
+	'ur': translate => translate( 'Urdu' ),
+	'uz': translate => translate( 'Uzbek' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **29** times:
`translate( 'Uzbekistan' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+	'te': translate => translate( 'Telugu' ),
+	'th': translate => translate( 'Thai' ),
+	'tir': translate => translate( 'Tigrinya' ),
+	'tl': translate => translate( 'Tagalog' ),
+	'tr': translate => translate( 'Turkish' ),
+	'tt': translate => translate( 'Tatar' ),
+	'ty': translate => translate( 'Tahitian' ),
+	'udm': translate => translate( 'Udmurt' ),
+	'ug': translate => translate( 'Uyghur' ),
+	'uk': translate => translate( 'Ukrainian' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Ukraine' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+	'su': translate => translate( 'Sundanese' ),
+	'sv': translate => translate( 'Swedish' ),
+	'ta': translate => translate( 'Tamil' ),
+	'te': translate => translate( 'Telugu' ),
+	'th': translate => translate( 'Thai' ),
+	'tir': translate => translate( 'Tigrinya' ),
+	'tl': translate => translate( 'Tagalog' ),
+	'tr': translate => translate( 'Turkish' ),
+	'tt': translate => translate( 'Tatar' ),
+	'ty': translate => translate( 'Tahitian' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **87** times:
`translate( 'Tahiti' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+	'si': translate => translate( 'Sinhala' ),
+	'sk': translate => translate( 'Slovak' ),
+	'sl': translate => translate( 'Slovenian' ),
+	'snd': translate => translate( 'Sindhi' ),
+	'so': translate => translate( 'Somali' ),
+	'sq': translate => translate( 'Albanian' ),
+	'sr': translate => translate( 'Serbian' ),
+	'su': translate => translate( 'Sundanese' ),
+	'sv': translate => translate( 'Swedish' ),
+	'ta': translate => translate( 'Tamil' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **11** times:
`translate( 'Tamil Nadu' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+	'qu': translate => translate( 'Quechua' ),
+	'ro': translate => translate( 'Romanian' ),
+	'ru': translate => translate( 'Russian' ),
+	'rup': translate => translate( 'Aromanian' ),
+	'sc': translate => translate( 'Sardinian' ),
+	'si': translate => translate( 'Sinhala' ),
+	'sk': translate => translate( 'Slovak' ),
+	'sl': translate => translate( 'Slovenian' ),
+	'snd': translate => translate( 'Sindhi' ),
+	'so': translate => translate( 'Somali' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **29** times:
`translate( 'Somalia' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+	'ne': translate => translate( 'Nepali' ),
+	'nl': translate => translate( 'Dutch' ),
+	'nn': translate => translate( 'Norwegian Nynorsk' ),
+	'no': translate => translate( 'Norwegian' ),
+	'non': translate => translate( 'Old Norse' ),
+	'nv': translate => translate( 'Navajo' ),
+	'oci': translate => translate( 'Occitan' ),
+	'or': translate => translate( 'Odia' ),
+	'os': translate => translate( 'Ossetic' ),
+	'pa': translate => translate( 'Punjabi' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **11** times:
`translate( 'Punjab' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+	'ml': translate => translate( 'Malayalam' ),
+	'mn': translate => translate( 'Mongolian' ),
+	'mr': translate => translate( 'Marathi' ),
+	'ms': translate => translate( 'Malay' ),
+	'mt': translate => translate( 'Maltese' ),
+	'mwl': translate => translate( 'Mirandese' ),
+	'nah': translate => translate( 'Nahuatl' ),
+	'nap': translate => translate( 'Neapolitan' ),
+	'nds': translate => translate( 'Low German' ),
+	'ne': translate => translate( 'Nepali' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Nepal' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+	'mk': translate => translate( 'Macedonian' ),
+	'ml': translate => translate( 'Malayalam' ),
+	'mn': translate => translate( 'Mongolian' ),
+	'mr': translate => translate( 'Marathi' ),
+	'ms': translate => translate( 'Malay' ),
+	'mt': translate => translate( 'Maltese' ),
+	'mwl': translate => translate( 'Mirandese' ),
+	'nah': translate => translate( 'Nahuatl' ),
+	'nap': translate => translate( 'Neapolitan' ),
+	'nds': translate => translate( 'Low German' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'German' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+	'la': translate => translate( 'Latin' ),
+	'li': translate => translate( 'Limburgish' ),
+	'lo': translate => translate( 'Lao' ),
+	'lt': translate => translate( 'Lithuanian' ),
+	'lv': translate => translate( 'Latvian' ),
+	'mk': translate => translate( 'Macedonian' ),
+	'ml': translate => translate( 'Malayalam' ),
+	'mn': translate => translate( 'Mongolian' ),
+	'mr': translate => translate( 'Marathi' ),
+	'ms': translate => translate( 'Malay' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Malaysia' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+	'ckb': translate => translate( 'Central Kurdish' ),
+	'kv': translate => translate( 'Komi' ),
+	'la': translate => translate( 'Latin' ),
+	'li': translate => translate( 'Limburgish' ),
+	'lo': translate => translate( 'Lao' ),
+	'lt': translate => translate( 'Lithuanian' ),
+	'lv': translate => translate( 'Latvian' ),
+	'mk': translate => translate( 'Macedonian' ),
+	'ml': translate => translate( 'Malayalam' ),
+	'mn': translate => translate( 'Mongolian' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Mongolia' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+	'ko': translate => translate( 'Korean' ),
+	'ks': translate => translate( 'Kashmiri' ),
+	'ckb': translate => translate( 'Central Kurdish' ),
+	'kv': translate => translate( 'Komi' ),
+	'la': translate => translate( 'Latin' ),
+	'li': translate => translate( 'Limburgish' ),
+	'lo': translate => translate( 'Lao' ),
+	'lt': translate => translate( 'Lithuanian' ),
+	'lv': translate => translate( 'Latvian' ),
+	'mk': translate => translate( 'Macedonian' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Macedonia' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+	'kn': translate => translate( 'Kannada' ),
+	'ko': translate => translate( 'Korean' ),
+	'ks': translate => translate( 'Kashmiri' ),
+	'ckb': translate => translate( 'Central Kurdish' ),
+	'kv': translate => translate( 'Komi' ),
+	'la': translate => translate( 'Latin' ),
+	'li': translate => translate( 'Limburgish' ),
+	'lo': translate => translate( 'Lao' ),
+	'lt': translate => translate( 'Lithuanian' ),
+	'lv': translate => translate( 'Latvian' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Latvia' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+	'kir': translate => translate( 'Kyrgyz' ),
+	'kk': translate => translate( 'Kazakh' ),
+	'km': translate => translate( 'Khmer' ),
+	'kn': translate => translate( 'Kannada' ),
+	'ko': translate => translate( 'Korean' ),
+	'ks': translate => translate( 'Kashmiri' ),
+	'ckb': translate => translate( 'Central Kurdish' ),
+	'kv': translate => translate( 'Komi' ),
+	'la': translate => translate( 'Latin' ),
+	'li': translate => translate( 'Limburgish' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **9** times:
`translate( 'Limburg' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 16]
```diff
@@ -0,0 +1,141 @@
...
+	'ka': translate => translate( 'Georgian' ),
+	'kir': translate => translate( 'Kyrgyz' ),
+	'kk': translate => translate( 'Kazakh' ),
+	'km': translate => translate( 'Khmer' ),
+	'kn': translate => translate( 'Kannada' ),
+	'ko': translate => translate( 'Korean' ),
+	'ks': translate => translate( 'Kashmiri' ),
+	'ckb': translate => translate( 'Central Kurdish' ),
+	'kv': translate => translate( 'Komi' ),
+	'la': translate => translate( 'Latin' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **10** times:
`translate( 'Latina' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 17]
```diff
@@ -0,0 +1,141 @@
...
+	'ia': translate => translate( 'Interlingua' ),
+	'id': translate => translate( 'Indonesian' ),
+	'ii': translate => translate( 'Sichuan Yi' ),
+	'ilo': translate => translate( 'Iloko' ),
+	'is': translate => translate( 'Icelandic' ),
+	'it': translate => translate( 'Italian' ),
+	'ja': translate => translate( 'Japanese' ),
+	'ka': translate => translate( 'Georgian' ),
+	'kir': translate => translate( 'Kyrgyz' ),
+	'kk': translate => translate( 'Kazakh' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Kazakhstan' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 18]
```diff
@@ -0,0 +1,141 @@
...
+	'hy': translate => translate( 'Armenian' ),
+	'ia': translate => translate( 'Interlingua' ),
+	'id': translate => translate( 'Indonesian' ),
+	'ii': translate => translate( 'Sichuan Yi' ),
+	'ilo': translate => translate( 'Iloko' ),
+	'is': translate => translate( 'Icelandic' ),
+	'it': translate => translate( 'Italian' ),
+	'ja': translate => translate( 'Japanese' ),
+	'ka': translate => translate( 'Georgian' ),
+	'kir': translate => translate( 'Kyrgyz' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Kyrgyzstan' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 19]
```diff
@@ -0,0 +1,141 @@
...
+	'hu': translate => translate( 'Hungarian' ),
+	'hy': translate => translate( 'Armenian' ),
+	'ia': translate => translate( 'Interlingua' ),
+	'id': translate => translate( 'Indonesian' ),
+	'ii': translate => translate( 'Sichuan Yi' ),
+	'ilo': translate => translate( 'Iloko' ),
+	'is': translate => translate( 'Icelandic' ),
+	'it': translate => translate( 'Italian' ),
+	'ja': translate => translate( 'Japanese' ),
+	'ka': translate => translate( 'Georgian' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Georgia' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 20]
```diff
@@ -0,0 +1,141 @@
...
+	'gn': translate => translate( 'Guarani' ),
+	'gu': translate => translate( 'Gujarati' ),
+	'he': translate => translate( 'Hebrew' ),
+	'hi': translate => translate( 'Hindi' ),
+	'hr': translate => translate( 'Croatian' ),
+	'hu': translate => translate( 'Hungarian' ),
+	'hy': translate => translate( 'Armenian' ),
+	'ia': translate => translate( 'Interlingua' ),
+	'id': translate => translate( 'Indonesian' ),
+	'ii': translate => translate( 'Sichuan Yi' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **8** times:
`translate( 'Sichuan' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 21]
```diff
@@ -0,0 +1,141 @@
...
+	'gl': translate => translate( 'Galician' ),
+	'gn': translate => translate( 'Guarani' ),
+	'gu': translate => translate( 'Gujarati' ),
+	'he': translate => translate( 'Hebrew' ),
+	'hi': translate => translate( 'Hindi' ),
+	'hr': translate => translate( 'Croatian' ),
+	'hu': translate => translate( 'Hungarian' ),
+	'hy': translate => translate( 'Armenian' ),
+	'ia': translate => translate( 'Interlingua' ),
+	'id': translate => translate( 'Indonesian' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Indonesia' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 22]
```diff
@@ -0,0 +1,141 @@
...
+	'ga': translate => translate( 'Irish' ),
+	'gd': translate => translate( 'Scottish Gaelic' ),
+	'gl': translate => translate( 'Galician' ),
+	'gn': translate => translate( 'Guarani' ),
+	'gu': translate => translate( 'Gujarati' ),
+	'he': translate => translate( 'Hebrew' ),
+	'hi': translate => translate( 'Hindi' ),
+	'hr': translate => translate( 'Croatian' ),
+	'hu': translate => translate( 'Hungarian' ),
+	'hy': translate => translate( 'Armenian' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Armenia' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 23]
```diff
@@ -0,0 +1,141 @@
...
+	'fr-be': translate => translate( 'Belgian French' ),
+	'fr-ca': translate => translate( 'Canadian French' ),
+	'fr-ch': translate => translate( 'Swiss French' ),
+	'fur': translate => translate( 'Friulian' ),
+	'fy': translate => translate( 'Western Frisian' ),
+	'ga': translate => translate( 'Irish' ),
+	'gd': translate => translate( 'Scottish Gaelic' ),
+	'gl': translate => translate( 'Galician' ),
+	'gn': translate => translate( 'Guarani' ),
+	'gu': translate => translate( 'Gujarati' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **13** times:
`translate( 'Gujarat' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 24]
```diff
@@ -0,0 +1,141 @@
...
+	'es-cl': translate => translate( 'Chilean Spanish' ),
+	'et': translate => translate( 'Estonian' ),
+	'eu': translate => translate( 'Basque' ),
+	'fa': translate => translate( 'Persian' ),
+	'fi': translate => translate( 'Finnish' ),
+	'fo': translate => translate( 'Faroese' ),
+	'fr': translate => translate( 'French' ),
+	'fr-be': translate => translate( 'Belgian French' ),
+	'fr-ca': translate => translate( 'Canadian French' ),
+	'fr-ch': translate => translate( 'Swiss French' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'French' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 25]
```diff
@@ -0,0 +1,141 @@
...
+	'es': translate => translate( 'Spanish' ),
+	'es-cl': translate => translate( 'Chilean Spanish' ),
+	'et': translate => translate( 'Estonian' ),
+	'eu': translate => translate( 'Basque' ),
+	'fa': translate => translate( 'Persian' ),
+	'fi': translate => translate( 'Finnish' ),
+	'fo': translate => translate( 'Faroese' ),
+	'fr': translate => translate( 'French' ),
+	'fr-be': translate => translate( 'Belgian French' ),
+	'fr-ca': translate => translate( 'Canadian French' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'French (Canada)' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 26]
```diff
@@ -0,0 +1,141 @@
...
+	'en': translate => translate( 'English' ),
+	'en-gb': translate => translate( 'British English' ),
+	'eo': translate => translate( 'Esperanto' ),
+	'es': translate => translate( 'Spanish' ),
+	'es-cl': translate => translate( 'Chilean Spanish' ),
+	'et': translate => translate( 'Estonian' ),
+	'eu': translate => translate( 'Basque' ),
+	'fa': translate => translate( 'Persian' ),
+	'fi': translate => translate( 'Finnish' ),
+	'fo': translate => translate( 'Faroese' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **85** times:
`translate( 'Faroe' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 27]
```diff
@@ -0,0 +1,141 @@
...
+	'as': translate => translate( 'Assamese' ),
+	'ast': translate => translate( 'Asturian' ),
+	'av': translate => translate( 'Avaric' ),
+	'ay': translate => translate( 'Aymara' ),
+	'az': translate => translate( 'Azerbaijani' ),
+	'ba': translate => translate( 'Bashkir' ),
+	'bel': translate => translate( 'Belarusian' ),
+	'bg': translate => translate( 'Bulgarian' ),
+	'bm': translate => translate( 'Bambara' ),
+	'bn': translate => translate( 'Bangla' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Bangladesh' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 28]
```diff
@@ -0,0 +1,141 @@
...
+	'am': translate => translate( 'Amharic' ),
+	'ar': translate => translate( 'Arabic' ),
+	'arc': translate => translate( 'Aramaic' ),
+	'as': translate => translate( 'Assamese' ),
+	'ast': translate => translate( 'Asturian' ),
+	'av': translate => translate( 'Avaric' ),
+	'ay': translate => translate( 'Aymara' ),
+	'az': translate => translate( 'Azerbaijani' ),
+	'ba': translate => translate( 'Bashkir' ),
+	'bel': translate => translate( 'Belarusian' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **29** times:
`translate( 'Belarus' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 29]
```diff
@@ -0,0 +1,141 @@
...
+	'af': translate => translate( 'Afrikaans' ),
+	'als': translate => translate( 'Alemannic' ),
+	'am': translate => translate( 'Amharic' ),
+	'ar': translate => translate( 'Arabic' ),
+	'arc': translate => translate( 'Aramaic' ),
+	'as': translate => translate( 'Assamese' ),
+	'ast': translate => translate( 'Asturian' ),
+	'av': translate => translate( 'Avaric' ),
+	'ay': translate => translate( 'Aymara' ),
+	'az': translate => translate( 'Azerbaijani' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **29** times:
`translate( 'Azerbaijan' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/language-picker/language-names.js [hunk 30]
```diff
@@ -0,0 +1,141 @@
+export default {
+	/* eslint-disable quote-props */
+	'af': translate => translate( 'Afrikaans' ),
+	'als': translate => translate( 'Alemannic' ),
+	'am': translate => translate( 'Amharic' ),
+	'ar': translate => translate( 'Arabic' ),
+	'arc': translate => translate( 'Aramaic' ),
+	'as': translate => translate( 'Assamese' ),
+	'ast': translate => translate( 'Asturian' ),
```
Comment (a8ci18n) [inline client/components/language-picker/language-names.js]: Hi! I've found a possible matching string that has already been translated **10** times:
`translate( 'Asturias' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by yoavf** [comment]
On wow, a8ci18e bot really got crazy about this.

Do we have a way to populate this from the CLDR? It'll be a bit silly to have to translate all the language names in all languages, while that information is already available in a library.

**Comment by jsnajdr** [comment]
> Do we have a way to populate this from the CLDR? It'll be a bit silly to have to translate all the language names in all languages, while that information is already available in a library.

Yes, we can populate the translations from CLDR with a simple script. I already generated the mappings from language code to English name using CLDR. I had to fix around 10 names manually, but the remaining ~130 ones can be automated.

**Comment by yoavf** [comment]
Ok, the question then is where/how to include this. We probably want to do it in the backend and add the translated language names to the language/translation file, rather than adding all language name in all locales to the main bundle.

**Comment by jsnajdr** [comment]
> Ok, the question then is where/how to include this. We probably want to do it in the backend and add the translated language names to the language/translation file, rather than adding all language name in all locales to the main bundle.

Adding the translations to the backend (i.e., the `widgets.wp.com/languages/calypso/*` files) with a CLDR script looks like the best option to me.

**Comment by alisterscott** [comment]
Closing as this PR is stale - please reopen if necessary

**Pull request closed without merging**